src/measures.tex

```diff
@@ -49,21 +47,22 @@ \subsection{Spaces of Measures}
 	\Lm_{r'}(S)_{A,\leq c}=\bigl\{\sum_{s\in S,n\in A}a_{n,s}T^n[s] \mid a_{n,s}\in \Z, \sum_{n\in A, s\in S}\lvert a_{n,s}\rvert r^n\leq c\bigr\}.
 \]
 This defines a profinite topology on $\Lm_{r'}(S)_{\leq c}$, for each $c>0$.
-
-\begin{definition}
-  \label{laurent-measures}
-  \uses{laurent-measures-fin}
-  \lean{laurent_measures}
-  \leanok
-  Let $0 < r' < 1$ be a real number, and let $S$ be a profinite set with a presentation $S=\varprojlim S_i$, where all $S_i$ are finite. For all $c>0$, let $\Lm_{r'}(S)_{\leq c}$ denote the projective limit
-  \[
-  \Lm_{r'}(S)_{\leq c}=\varprojlim\Lm_{r'}(S_i)_{\leq c},
-  \]
-  endowed with the projective limit topology, and set
-  \[
-  \Lm_{r'}(S)=\varinjlim_{c > 0}\Lm_{r'}(S)_{\leq c}.
-  \]
 \end{definition}
+Passing to the limit, we obtain the Laurent measures on an arbitrary profinite set $S$ as follows. 
+%\begin{definition}
+% \label{laurent-measures}
+% \lean{laurent_measures}
+% \leanok
+%  Let $0 < r' < 1$ be a real number, and let $S$ be a profinite set with 
+Starting with a presentation $S=\varprojlim S_i$, where all the $S_i$ are finite, define for all $c>0$ the space $\Lm_{r'}(S)_{\leq c}$ denote the limit
```

Comment:
This sentence is a bit mangled. The "define" and "denote" aren't mixing well with each other.